contrib/buildahimage/testing/Containerfile

```
@@ -0,0 +1,54 @@
+# testing/Containerfile
+#
+# Build a Buildah container image from the latest
+# stable version of Buildah on the Fedoras Updates System.
+# https://bodhi.fedoraproject.org/updates/?search=buildah
+# This image can be used to create a secured container
+# that runs safely with privileges within the container.
+#
+FROM registry.fedoraproject.org/fedora:latest
+
+# Don't include container-selinux and remove
+# directories used by dnf that are just taking
+# up space.
+RUN useradd build; \
+    dnf -y update && \
+    rpm --setcaps shadow-utils 2>/dev/null && \
+    dnf -y install buildah fuse-overlayfs \
+        --exclude container-selinux && \
+        --enablerepo updates-testing && \
```

If you don't change back to yum, I think you need to add an equal sign here to keep dnf happy

```suggestion
        --enablerepo=updates-testing && \
```

---

I don't believe this matters, but I added it.